client/src/components/ScorecardMapSearchBar.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class='searchbar-container'>
+    <div class='zoom-options'>
+      <search-bar-option
+        v-for='option in options'
+        :key='option'
+        :text-content='option'
+        :selected='getSelected(option)'
+        @click='setSelected(option)' />
+    </div>
+    <map-geocoder-wrapper
```

Comment:
Added a TODO for this for now. This is a bit more complex than it seems because the input doesn't automatically stretch to fit a longer placeholder, so you have to dynamically set width which is a bit hacky and apparently inconsistent across browsers: https://stackoverflow.com/questions/17302794/how-to-set-an-input-width-to-match-the-placeholder-text-width.

Since it's such a small thing I think we should hold off on introducing this logic. An alternative solution would be to switch to what the [mocks](https://www.figma.com/file/0LGxjZnPD3N0dCqHv8LRr6/NW-Map-Design-System-WIP?node-id=452%3A5994) show which is show a prompt to redo the search. I think the mock is supposed to reroute back to the landing page so we held off because it seems like going back and forth between pages would introduce unnecessary latency. We can discuss this with Nicole.